src/main/java/com/example/shorturl/service/ShortenUrlService.java

```diff
@@ -67,7 +66,7 @@ public UrlEntity getEncodedUrl(String url){
         return  urlList.get(0);
```

Comment:
Let's think about better approach and save in this method itself

Reply:
Checked